src/Microsoft.AspNetCore.Authentication.OAuth/Claims/ClaimActionCollectionMapExtensions.cs

```diff
@@ -87,6 +87,15 @@ public static void MapCustomJson(this ClaimActionCollection collection, string c
             collection.Add(new CustomJsonClaimAction(claimType, valueType, resolver));
         }
 
+        /// <summary>
+        /// Maps all values from the json user data as claims, excluding duplicates.
+        /// </summary>
+        /// <param name="collection"></param>
+        public static void MapAllClaims(this ClaimActionCollection collection)
```

Comment:
Shorten to MapAll to be consistent with all the other methods that don't have Claims suffix?